docs/Developers-Guide.md

```diff
@@ -77,16 +95,59 @@ The unittests for the api client can use ```mock``` to emulate responses from th
 
 To introduce a new API endpoint to be tested, the endpoint needs to be registered in the ```url_router``` section in ```/api_client/python/timesketch_api_client/test_lib.py``` and the response needs to be defined in the same file.
 
+#### end2end tests
+
+End2end (e2e) tests are used at Github with every commit. Those tests will setup and run with a real elasticsearch and data imported. 
+To run the e2e-tests locally execute to setup the e2e docker images and run them:
+
+```bash
+sh end_to_end_tests/tools/run_in_container.py
+```
+
+The tests are stored in:
+
+```shell
+end_to_end_tests/*.py
+```
+
+And the sample data (currently a plaso file and a csv) is stored in:
+
+```shell
+end_to_end_tests/test_data/
+```
+
+#### Writing end2end tests
+
+While writing end2end tests it is recommended to create a symling to the source files.
```

Comment:
`symling to ` ? are you referring to a simlink, and why simlink instead of copy to the folder

Reply:
ah... I read below... 

I would rather say `recommended` you can say `one approach to make it easier to develop these e2e tests is to create a simlink to the source files, in order for the changes to the files being reflected in the container`

Another way would be to simply mount the TS source code to this folder, the /usr/local/lib/pytho.....

Reply:
Good point, will add that